src/PanelTraits/Query.php

```diff
@@ -53,6 +53,23 @@ public function orderBy($field, $order = 'asc')
         return $this->query->orderBy($field, $order);
     }
 
+    /**
+     * Order the results of the query in a custom way.
+     */
+    public function customOrderBy($column, $column_direction)
```

Comment:
We could probably type hint these now.

Reply:
Can you be more specific here? looking at other methods I'm not sure what kind of type hinting I can give.